app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
+	
+Here's an example:
+
+[See it on Plunker](http://plnkr.co/edit/VPktVlI0PWMnYdYVsVBx?p=preview)
+
+```html
+<!-- Create a template for the custom element -->
+<dom-module id='custom-element'>
+  <template>
+    <h1>Heading!</h1>
+    <p>We are elements in custom-element's local DOM.</p>
+  </template>
+</dom-module>
+
+<!-- Drop the custom element on the page -->
+<custom-element></custom-element>
+
+<!-- Register the element -->
+<script>
+  class CustomElement extends Polymer.Element {
+    static get is() {
+      return "custom-element";
+    }
+  }
+  customElements.define(CustomElement.is, CustomElement);
+</script>
+```
+
+The HTML elements in your template become children within your custom element's shadow DOM. Shadow DOM provides a mechanism for encapsulation, meaning that elements in side the shadow DOM are not accessible to styling rules external to the shadow DOM. 
```

Comment:
within => in
in side => inside
"not accessible to styling" Might want to say "don't match selectors outside the shadow DOM" or say what you did, but then add, "Specifically, elements inside a shadow tree don't match selectors specified outside that shadow tree—in the main document, or in another shadow tree."

does that make sense?